modules/pages/partials/travel-app-data-model.adoc

```diff
@@ -19,16 +19,16 @@ The model for each kind of document contains:
 * A `type` field that identifies the kind of document
 
 The following figure illustrates the relationship between the different kinds of documents in the Travel App.
-It shows the primary key, ID, and type fields that each document has, plus a few representative fields in each type of document.
+It shows the primary key, ID, and type fields that each document has, plus some other representative fields in each type of document.
 
-.Documents in the Travel App data model
+.Documents in the Travel Application data model
 image::travel-app-data-model.png[Travel Sample data model diagram]
 
-Landmark documents are not used in the travel-sample application, which focuses on a more specific subset of point of interests that have their own type: hotels.
+Landmark documents aren't used in the travel-sample application, which focuses on a more specific subset of point of interests that have their own type: hotels.
```

Comment:
I feel this sort of change dilutes the meaning (through de-emphasis) a little - perhaps the style guide item needs revisiting?

Reply:
I'll add it to the agenda for the next style meeting. I think there is some incentive to remove mandatory contractions.